src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -333,6 +330,30 @@ internal ECDsa CreateECDsa(string algorithm, bool usePrivateKey)
             if (Y == null)
                 throw LogHelper.LogArgumentNullException(nameof(Y));
 
+#if !WINDOWS && NETSTANDARD2_0
```

Comment:
Do ECParameters exist in Core 2.0 on Windows?
If so could this code work on Windows?

Reply:
The issue seems to be that .net 4.6.1 does not support ECParameters so our target 4.6.1 would start failing on Windows, hence Windows needs to be excluded.